src/openface_node.py

```diff
@@ -18,7 +18,23 @@
 from face_client import FaceClient
 
 from timeout import Timeout
+import pickle
+import rospkg
 
+rp = rospkg.RosPack()
+_OPENFACE_DIR = rp.get_path('openface_ros') # find openface (should be at same level as openface_ros per install instructions)
+_OPENFACE_DIR = _OPENFACE_DIR[:-4] + "/" 
+if (not os.path.exists( _OPENFACE_DIR ) ):
+  print "openface directory does not exist; exiting"
+  sys.exit()
+
+_DEMO_LIB_DIR = rp.get_path('hsr_demo_library') + "/data/"
+if (not os.path.exists( _DEMO_LIB_DIR ) ):
+  print "demo lib directory does not exist; exiting"
+  sys.exit()
+
+_FACE_DICT_FNAME = 'toyota_faces.pickle'
+_SAVE = False
```

Comment:
Ok, now using rosparam a lot; similarly prints are now rospy.loginfo's.

Re: the pickle file, if no pickle file is passed in, we're back to the original logic.

I also removed dead / commented out code.